src/utils.js

```diff
@@ -1,12 +1,21 @@
 // @flow
-import { uniq, memoize } from 'lodash'
-import type { Component, IndividualSides, MarginSizes } from './types'
-import marginStyle from './margin.css'
+import { memoize, compact } from 'lodash'
+import type { Spacing, Component } from './types'
+import style from './spacing.css'
 
 /* eslint-disable no-unused-vars */
 const noop = (...params: any[]) => null
 /* eslint-enable no-unused-vars */
 const isProd = process.env.NODE_ENV === 'production'
+const stringify = param => JSON.stringify(param)
```

Comment:
nice, I thought I would have to bind to JSON but I didn't try it, this looks cleaner, will update